src/common-principles.md

```diff
@@ -536,12 +547,38 @@ like in the example below.
             "F": {
                 "Description": "Female",
                 "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
-            },
+            }
         }
     }
 }
 ```
 
+### Compressed tabular files
+
+Large tabular information such as physiological recordings MAY be stored with
+[compressed tab-delineated (TSVGZ) files](glossary.md#tsvgz-extensions).
+Rules for formatting plain-text tabular files apply to TSVGZ files with three exceptions:
+
+1.  The contents of TSVGZ files SHOULD be compressed with
+    [gzip](https://datatracker.ietf.org/doc/html/rfc1952).
+1.  Compressed tabular files SHOULD NOT contain a header in the first row
+    indicating the column names.
+1.  TSVGZ files SHOULD be accompanied by a JSON file with the same name as their
+    corresponding tabular file but with a `.json` extension.
+
+???+ warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."
+
+    In contrast to plain-text TSV files, compressed tabular files files
+    MUST NOT include a header line.
+    Column names MUST be specified in the JSON file following the
+    [`Columns` metadata](glossary.md#columns-metadata) specifications.
+    As plain-text tabular data, column names MUST NOT be blank (that is, an empty string),
+    and MUST NOT be duplicated within a single JSON file describing a TSVGZ file.
```

Comment:
I'm confused. This seems to contradict the previous SHOULD lines. What am I missing?

Reply:
I believe this was in the physio section meaning that you should not define the same column twice in a JSON object (which is a bit redundant because you can't have the same key twice). Happy to rewrite or remove -- what do you prefer?

Reply:
Ah. I would say it just once unless there is a specific exception or caveat here. Otherwise it's unclear why you're saying it twice with slightly different words.

Apologies, I'm not feeling my best, so if what I say doesn't make sense, feel free to ignore me.

Reply:
Sad to hear you are not feeling great :( However, this is definitely confusing in the first place and is confusing to me, too. The current docs read:

https://github.com/bids-standard/bids-specification/blob/bd08602760d2fa0fd61e8f579a2677ea9bda8588/src/modality-specific-files/physiological-and-other-continuous-recordings.md#L75-L77

I just moved this here. I'll try to check if this is duplicated from the standard tabular files and remove if so.

Reply:
Edited to make it clear that this comes from the plain text specs: https://github.com/bids-standard/bids-specification/pull/1749/commits/3811a8c4262a3ef32d7f37e5675ffdc5f41351c6

Reply:
Marking resolved -- please suggest again if the direction I took is not satisfactory.